lms/djangoapps/grades/tests/data/capa.xml

```diff
@@ -0,0 +1,6 @@
+<problem url_name="capa-optionresponse">
```

Comment:
Are these test fixtures specific to grades testing?  If not, should they belong in the common test/data directory?  Or are we moving away from using common test xml data?